src/discord.cpp

```diff
@@ -54,47 +107,43 @@ void Discord::handleDiscordError(int errorCode, const char* message)
 
 void Discord::handleDiscordJoinGame(const char* joinSecret)
 {
-
     qDebug() << "Discord handleDiscordJoinGame!";
 }
 
 void Discord::handleDiscordSpectateGame(const char* spectateSecret)
 {
-
     qDebug() << "Discord handleDiscordSpectateGame!";
 }
 
 void Discord::handleDiscordJoinRequest(const DiscordUser* request)
 {
-
     qDebug() << "Discord handleDiscordJoinRequest!";
 }
 
 void Discord::UpdatePresence()
 {
     char buffer[256];
-    DiscordRichPresence discordPresence;
-    memset(&discordPresence, 0, sizeof(discordPresence));
-//    discordPresence.state = "In a raiding party";
-//    sprintf(buffer, "Playing Achaea");
-//    discordPresence.details = buffer;
-//    discordPresence.startTimestamp = time(0);
-////    discordPresence.smallImageKey = "achaea";
-//    discordPresence.largeImageKey = "achaea";
-//    discordPresence.largeImageText = "Dragon (level 100)";
-
-    discordPresence.state = "Raiding party";
+    memset(&mDiscordPresence, 0, sizeof(mDiscordPresence));
+    //    mDiscordPresence.state = "In a raiding party";
+    //    sprintf(buffer, "Playing Achaea");
+    //    mDiscordPresence.details = buffer;
+    //    mDiscordPresence.startTimestamp = time(0);
+    ////    mDiscordPresence.smallImageKey = "achaea";
+    //    mDiscordPresence.largeImageKey = "achaea";
+    //    mDiscordPresence.largeImageText = "Dragon (level 100)";
+
+    mDiscordPresence.state = "Raiding party";
     sprintf(buffer, "Playing Luminari");
-    discordPresence.details = buffer;
-    discordPresence.startTimestamp = time(0);
-//    discordPresence.smallImageKey = "achaea";
-    discordPresence.largeImageKey = "luminari";
-    discordPresence.largeImageText = "Dragon (level 100)";
-
-    discordPresence.partySize = 5;
-    discordPresence.partyMax = 6;
-    discordPresence.matchSecret = "zdgfghrfsyheqrwqgshbfxdq35 4 5";
-    discordPresence.joinSecret = "ASFDFSHR512345 RASGSADWr";
-    discordPresence.instance = 1;
-    Discord_UpdatePresence(&discordPresence);
+    mDiscordPresence.details = buffer;
+    mDiscordPresence.startTimestamp = time(0);
+    //    mDiscordPresence.smallImageKey = "achaea";
+    mDiscordPresence.largeImageKey = "luminari";
+    mDiscordPresence.largeImageText = "Dragon (level 100)";
+
+    mDiscordPresence.partySize = 5;
+    mDiscordPresence.partyMax = 6;
+    mDiscordPresence.matchSecret = "zdgfghrfsyheqrwqgshbfxdq35 4 5";
+    mDiscordPresence.joinSecret = "ASFDFSHR512345 RASGSADWr";
```

Comment:
Can we have an explanation somewhere about what these magic numbers mean/are for - or even the whole `mDiscordPresence`?